tutorials/SingleQubitGates/SingleQubitGates.csproj

```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
```

Comment:
Should line 3 here be updated to `netcoreapp3.0` as well? This seems to fail CI (Sorry, GitHub doesn't let me leave a comment there)

Reply:
yeap. this project was added after the migration of the rest was done so it was not picked up.
I'll update.